frontend/src/pages/ForgotPassword.spec.js

```diff
@@ -162,23 +152,9 @@ describe('ForgotPassword', () => {
             it('button link redirects to "/login"', () => {
               expect(wrapper.find('.test-message-button').attributes('href')).toBe('/login')
             })
-
-            it.skip('click redirects to "/login"', () => {
-              // expect(mockRouterPush).toBeCalledWith('/login')
-            })
           })
         })
       })
     })
-
-    describe('comingFrom login', () => {
-      beforeEach(() => {
-        wrapper = Wrapper(createMockObject('resetPassword'))
-      })
-
-      it('has another subtitle', () => {
-        expect(wrapper.find('p.text-lead').text()).toEqual('settings.password.resend_subtitle')
-      })
-    })
```

Comment:
Why this is removed?

The code to extract the comming from is still in `ForgotPassword.vue`.

Reply:
'.text-lead' no longer exists in the file ForgotPassword.js